src/oop_ext/interface/_interface.py

```diff
@@ -191,12 +191,16 @@ def _IsClass(obj):
     return isinstance(obj, type)
 
 
-def IsImplementation(class_or_instance, interface):
+def IsImplementation(class_or_instance, interface, *, requires_declaration=False):
     """
     :type class_or_instance: type or classobj or object
 
     :type interface: Type[Interface]
 
+    :type requires_declaration: bool
+        If `True`, the Interface must have been explicitly declared through :py:func:`ImplementsInterface`
```

Comment:
Also mention here the function that lets you declare that an implementation implements a given interface externally.

Reply:
Done